spec/content/spec.yml

```diff
@@ -46,6 +46,11 @@ spec:
     name: kibana
     required: false
     $ref: "./kibana/spec.yml"
+  - description: Folder containing Index data assets used by the package
+    type: folder
+    name: index_data
+    required: false
+    $ref: "./index_data/spec.yml"
```

Comment:
@mrodm we are adding a new "index_data" directory at the root level, for collections of documents that should be installed on package installation. By now the only kind of collection we support would be "knowledge bases" for AI assistants. Other collections we could add in the future could be sample data for example.

Do you foresee any problem with this approach or this new directory at the root level?

Reply:
Yeah, we would only need a change in the registry if this were going to be used for discovery of packages. Let's see how the support in Kibana progresses.